tutorials/fundamentals/array-to-string.md

```diff
@@ -0,0 +1,37 @@
+
+Javascript's handy way of converting arrays to strings is the
+`.toString()` method. Simply call it on the array and it will
+return the entries in the form of a string, each separated by
+a comma as seen below:
+
+```javascript
+let arr = ['Dog','Cat','Fish','Bread'];
+arr.toString(); // prints Dog,Cat,Fish,Bread
+```
+
+## + Operator
```

Comment:
Let's get rid of the `+` operator section, this isn't something we should recommend. `join()` is better.